fetcher/jvn/jvn.go

```diff
@@ -65,26 +66,21 @@ func FetchJVN() (models.FetchedCPEs, error) {
 	return fetched, nil
 }
 
-func makeJvnURLs(years []int) (urls []string) {
-	latestFeeds := []string{
-		"https://jvndb.jvn.jp/ja/rss/jvndb_new.rdf",
-		"https://jvndb.jvn.jp/ja/rss/jvndb.rdf",
-	}
-
-	if len(years) == 0 {
-		return latestFeeds
-	}
-
+func makeJvnURLs() (urls []string, err error) {
 	urlFormat := "https://jvndb.jvn.jp/ja/rss/years/jvndb_%d.rdf"
-	for _, year := range years {
-		urls = append(urls, fmt.Sprintf(urlFormat, year))
 
-		thisYear := time.Now().Year()
-		if year == thisYear {
-			urls = append(urls, latestFeeds...)
-		}
+	thisYear, err := strconv.Atoi(time.Now().Format("2006"))
```

Comment:
Any reason to change from `time.Now().Year()` ?

Reply:
https://github.com/vulsio/go-cpe-dictionary/pull/216/commits/c789463945627f90561b947ffe5c1f50f4670fa4